base/show.jl

```diff
@@ -61,6 +61,11 @@ The following properties are in common use:
  - `:displaysize`: A `Tuple{Int,Int}` giving the size in rows and columns to use for text
    output. This can be used to override the display size for called functions, but to
    get the size of the screen use the `displaysize` function.
+ - `:typeinfo`: a type specification characterizing the information already expressed
```

Comment:
"type specification" is a bit vague. Maybe "A `Type`"? "expressed" could be "printed".